hyperspy/samfire_utils/samfire_pool.py

```diff
@@ -51,8 +51,8 @@ class SamfirePool(ParallelPool):
         given SAMFire object, populates the workers with the required
         information. In case of multiprocessing, starts worker listening to the
         queues.
-    update_optional_names
-        updates the optional names in all workers
+    update_parameters
```

Comment:
However, it'll be good to describe in the header and possibly the changes list just in case somebody actually cares...